### PR TITLE
Add Secure auth store example

### PR DESCRIPTION
**Pull request by totzk9**
An example that provides makes the auth persistent and secure than `SharedPreferences`

### CODE REVIEW

**Comment by dbarrosop** [comment]
Amazing work, thanks. I will ask a colleague to review in the upcoming days.

File: packages/nhost_dart/pubspec.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ dependencies:
   nhost_graphql_adapter: ^4.0.5
 dev_dependencies:
   fake_async: ^1.3.1
-  graphql: ^5.1.3
+  graphql: ^5.2.0-beta.9
```
Comment (onehassan) [inline packages/nhost_dart/pubspec.yaml:26]: Is the version upgrade to `graphql: ^5.2.0-beta.9` required for the this example ?

Comment (totzk9) [reply packages/nhost_dart/pubspec.yaml:26]: No. I messed up the merging of main branch

**Comment by totzk9** [comment]
Closed this PR as being replaced with https://github.com/nhost/nhost-dart/pull/154 as this branch was messed up

**Pull request closed without merging**